src/billing/components/PayAsYouGo/PlanTypePanel.tsx

```diff
@@ -58,10 +50,13 @@ const PlanTypePanel: FC = () => {
             <h5>Last Update</h5>
           </Panel.Header>
           <Panel.Body size={ComponentSize.ExtraSmall}>
-            {new Date(billingInfo.balanceUpdatedAt).toLocaleString(
-              'default',
-              TimeOptions
-            )}{' '}
+            {new Date(billingInfo.balanceUpdatedAt).toLocaleString('default', {
+              month: 'numeric',
+              day: 'numeric',
+              year: 'numeric',
+              hour: 'numeric',
+              minute: '2-digit',
```

Comment:
nit: I kinda like the `const TimeOptions` being pulled out because now this object will be re-created on every render. But meh, it does read nicer so I'm fine with it either way 😅

Reply:
Truthfully I only changed it because it was giving me a TS error :(